source/loader/ur_libapi.cpp

```diff
@@ -35,7 +35,7 @@ extern "C" {
 ///     - ::UR_RESULT_ERROR_UNINITIALIZED
 ///     - ::UR_RESULT_ERROR_DEVICE_LOST
 ///     - ::UR_RESULT_ERROR_INVALID_ENUMERATION
-///         + `0x1f < device_flags`
+///         + `0xffffffe0 & device_flags`
```

Comment:
Shouldn't this be using the name of the mask macro?

Reply:
Fixed. It's the same value, but makes it more readable. 👍